tests/model_forms/tests.py

```diff
@@ -1789,6 +1717,49 @@ class WriterForm(forms.Form):
         self.assertEqual(len(params), 1)
 
 
+class MultipleChoiceWithFieldNameTests(TestCase):
+
+    def test_to_field_name_17657(self):
+        """
+        #17657 -- Make sure ModelForm respects to_field_name attribute of ModelMultipleChoiceField
```

Comment:
chop "Make sure" in favor of simply stating the expected behavior. The ticket reference isn't really needed -- we reserve that for obscure issues.